src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj

```diff
@@ -7,6 +7,7 @@
 
     <PathToMsalSources>$(MSBuildThisFileDirectory)../Microsoft.Identity.Client/</PathToMsalSources>
     <UseWindowsForms>true</UseWindowsForms>
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
Apparently this is needed with 1.0.864.35; otherwise on NetFx even if referencing MSAL project directly, WV2 wouldn't work.